botorch/optim/optimize_mixed.py

```diff
@@ -573,12 +686,12 @@ def optimize_acqf_mixed_alternating(
     `options.get("max_discrete_values", MAX_DISCRETE_VALUES)` values will
     be optimized using continuous relaxation.
 
-    # TODO: Support categorical variables.
-
     Args:
         acq_function: BoTorch Acquisition function.
         bounds: A `2 x d` tensor of lower and upper bounds for each column of `X`.
         discrete_dims: A list of indices corresponding to integer and binary parameters.
+        cat_dims: A list of indices corresponding to categorical parameters.
+            If `None`, no categorical parameters are assumed.
```

Comment:
Seems unnecessary, and we don't have this in the docstring for `discrete_dims` either
 ```suggestion
```